src/patternfly/components/MultipleFileUpload/multiple-file-upload.scss

```
@@ -54,46 +62,55 @@
   --#{$multiple-file-upload}--m-horizontal__main--GridTemplateAreas:
     "title upload"
     "info upload";
-  --#{$multiple-file-upload}--m-horizontal__main--Gap: var(--#{$pf-global}--spacer--sm) var(--#{$pf-global}--spacer--2xl);
-  --#{$multiple-file-upload}--m-horizontal__main--PaddingBottom: var(--#{$pf-global}--spacer--lg);
+  --#{$multiple-file-upload}--m-horizontal__main--Gap: var(--pf-t--global--spacer--sm) var(--pf-t--global--spacer--lg);
+  --#{$multiple-file-upload}--m-horizontal__main--PaddingBottom: var(--pf-t--global--spacer--lg);
   --#{$multiple-file-upload}--m-horizontal__title--GridTemplateColumns: auto 1fr;
-  --#{$multiple-file-upload}--m-horizontal__title--Gap: var(--#{$pf-global}--spacer--xs);
+  --#{$multiple-file-upload}--m-horizontal__title--Gap: var(--pf-t--global--spacer--sm);
+  --#{$multiple-file-upload}--m-horizontal__title-icon--FontSize: var(--pf-t--global--font--size--md);
+  --#{$multiple-file-upload}--m-horizontal__title-text--FontSize: var(--pf-t--global--font--size--md);
```

looks like this should use the xs heading token?

```suggestion
  --#{$multiple-file-upload}--m-horizontal__title-text--FontSize: var(--pf-t--global--font--size--heading--xs);
```